pytest_html/plugin.py

```diff
@@ -149,13 +149,16 @@ def _appendrow(self, result, report):
         if report.when != 'call':
             test_id = '::'.join([report.nodeid, report.when])
 
+        logs_attribs = {"data-type-associated" : result, 
+                        "data-test-associated" : test_id};
```

Comment:
Could we use `data-test-result` and `data-test-id` instead? Also, let's make the result lowercase.